README.md

```diff
@@ -17,28 +17,32 @@
 ## Overview
 
 ### Who we are
-We are the Technical Research Applications Development group found within the [Arizona Institute for Resilience (AIR)](https://air.arizona.edu). We are a group of technology professionals and students who provide a variety of technical services and collaborations to the [Centers and Programs](https://air.arizona.edu/centers-programs) found within AIR. Our project collaborations span the globe and make meaningful impacts from the desert southwestern United States to the Middle East and places in between. AIR explores and develops solutions with campus and community partners that serve human and natural communities on a global scale by engaging a full array of disciplines, professional schools, international capacity, and entrepreneurial opportunities.
+The Technical Research Applications Development (TRAD) group is found within the [Arizona Institute for Resilience (AIR)](https://air.arizona.edu). TRAD is a group of technology research professionals and students who provide a variety of technical research services and collaborations to the [Centers and Programs](https://air.arizona.edu/centers-programs) found within AIR. Our project collaborations span the globe and make meaningful impacts from the desert southwestern United States to the Middle East and places in between.
 
```

Comment:
```suggestion
The Technical Research Applications Development (TRAD) is a group of technology research professionals and students who provide technical research services and collaborations to the [Centers and Programs](https://air.arizona.edu/centers-programs) found within within the [Arizona Institute for Resilience (AIR)](https://air.arizona.edu). Our project collaborations span the globe and make meaningful impacts from the desert southwestern United States to the Middle East and places in between.

```
 Maybe a nit, but I felt like the first sentence didn't say anything previously. If we're trying to answer the question "What is TRAD?" answering with "TRAD is in AIR" first is a little strange. I think this rework gets to the point a little faster.  What do you think?